OracleWebLogic/samples/12213-webtier-apache/container-scripts/run-httpd.sh

```diff
@@ -6,13 +6,19 @@
 #
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 #
-# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2018-2020 Oracle and/or its affiliates.
 #
-# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 # Apache gets grumpy about PID files pre-existing
 rm -f /run/httpd/httpd.pid
 
+# Inorder to listen the apache container on port 8080, the variable NonPriviledgedPorts to be set to true.
```

Comment:
Inorder to listen the apache -> To listen to the Apache
NonPriviledgedPorts to be set -> NonPriviledgedPorts must to be set